rasa/core/policies/rule_policy.py

```diff
@@ -47,13 +52,14 @@ def supported_data() -> SupportedData:
         Returns:
             The data type supported by this policy (rule data).
         """
-        return SupportedData.RULE_DATA
+        return SupportedData.ML_AND_RULE_DATA
 
     def __init__(
         self,
         featurizer: Optional[TrackerFeaturizer] = None,
         priority: int = FORM_POLICY_PRIORITY,
         lookup: Optional[Dict] = None,
+        negative_lookup: Optional[Dict] = None,
```

Comment:
please add a docstring which describes the old and new parameters

Reply:
forgot to remove `negative_lookup`
added docstring to memoization policy, since the params are the same